Assets/Scripts/Interactables/Doohickey.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Doohickey : Interactable
+{
+  public float interactionForce = 10f;
+
+  protected override void MouthInteraction()
+  {
+    base.MouthInteraction();
+    MoveUpwards();
+  }
```

Comment:
We'll need to override any of the Interactions we want to use
then call the method of the same name in the `base` class.
Finally we can add whatever logic makes this unique.
Playing audio will be handled by the base class.

We don't need to override an interaction if we don't intend to use it, though we haven't added in any logic to guard those interactions in the base class yet. soon!